Shop_Unity_base/Assets/Scripts/Shop/Model/Customer.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Model
+{
+    using System;
+    using System.Collections.Generic;
+    using Utility;
+
+    public class Customer
```

Comment:
Since the Customer is in the Model namespace and operates as such, would it not be sensible to create its own View and Controller too? I am unsure of your requirements and how this suggestion fits within them.

Reply:
I am unsure either, but I have thought of that, certainly. At least I'd like to have a ShopController and CustomerController. Same on items, give them a Drawable component (when I put in components)